warn/warn_cosmetic.go

```diff
@@ -124,80 +124,104 @@ func loadOnTopWarning(f *build.File, fix bool) []*Finding {
 	return findings
 }
 
-func outOfOrderLoadWarning(f *build.File, fix bool) []*Finding {
-	findings := []*Finding{}
+// compareLoadLabels compares two module names
+func compareLoadLabels(load1Label, load2Label string) bool {
+	// handle absolute labels with explicit repositories separately to
+	// make sure they preceed absolute and relative labels without repos
+	isExplicitRepo1 := strings.HasPrefix(load1Label, "@")
+	isExplicitRepo2 := strings.HasPrefix(load2Label, "@")
+	if isExplicitRepo1 == isExplicitRepo2 {
+		// Either both labels have explicit repository names or both don't, compare their packages
+		// and break ties using file names if necessary
 
-	if f.Type == build.TypeWorkspace {
-		// Not applicable for WORKSPACE files
-		return findings
-	}
+		module1Parts := strings.SplitN(strings.TrimLeft(load1Label, "@"), ":", 2)
```

Comment:
Is it useful to trim "@"?

Reply:
I've just fixed the code written by someone else, but it seems that it can be removed.